hw/ip/prim/lint/prim.waiver

```diff
@@ -16,6 +16,10 @@ waive -rules {LINE_LENGTH} -location {prim_assert.sv} -msg {Line length of} \
 waive -rules INTEGER           -location {prim_packer.sv} -msg {'i' of type int used as a non-constant value} \
       -comment "This assigns int i (signed) to a multibit logic variable (unsigned), which is fine"
 
+# prim_packer_fifo
+waive -rules {ZERO_REP} -location {prim_packer_fifo.sv} -msg {Replication count is zero in '{{DepthW{1'b0}},1'b1}}' \
+      -comment "If InW equals OutW, DepthW is zero"
+
```

Comment:
You are absolutely right @mwbranstad, this waiver rule should be put in the prim.waiver and not the edn.waiver. That's now done. I would appreciate if you could give this a try with AscentLint. I don't know if I got the syntax right or if this will completely mess up the nightly lint run.

Reply:
"Would you mind trying that out please @mwbranstad ?"

I am going to try, so cross your figures.